examples/basic-bidding-sale/basic-bidding-sale.repl

```diff
@@ -190,23 +214,43 @@
 
   (env-data {
     "bid-id": (get-bid-id "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk" "k:bidder")
-    ,"market-guard": {"keys": ["market"], "pred": "keys-all"}
-    ,"seller-guard": {"keys": ["account"], "pred": "keys-all"}
     ,"sale-id": "DKc5HEWcmP8iPWue2WvMcnjn_WnowmuHz5Ogukm2SWk"
   })
 
-  (marmalade-v2.abc.create-account "k:market" (read-keyset 'market-guard))
-
   (env-sigs
-  [{'key:'account
-    ,'caps: [
-      (UPDATE_QUOTE_GUARD (read-msg 'sale-id))
-      (SELLER (read-msg 'sale-id))
-    ]}
+    [{'key:'account
+      ,'caps: []}
   ])
 
   (expect "Accepting bid succeeds"
     true
     (accept-bid (read-msg "bid-id") (read-msg "sale-id"))
   )
 (commit-tx)
+
+(begin-tx "Finalize sale")
+  (env-hash (hash "finalize-sale-0"))
+  (use marmalade-v2.quote-manager)
+  (use marmalade-v2.ledger)
+  (use free.basic-bidding-sale)
+
+  (env-data {
```

Comment:
In our sale flow, isn't the auction's escrow account the `buyer_fungible_account`?

Reply:
No, actually in our sale flow the `buyer_fungible_account` isn't being used because the funds have already been transferred from the acution's escrow account when we called the `reserve-sale` function.

`buyer_fungible_account` is used in a 'regular' quoted sale flow, and stands for the fungible account that holds the funds to buy.

Reply:
Got it, then it seems like `buyer-fungible-account` key is unnecessary at this step. Should we have this(https://github.com/kadena-io/marmalade/blob/v2/pact/policy-manager/policy-manager.pact#L339) code block  let block and not require reserved sales to proveide `buyer-fungible-account` in env-data?